src/visuals.js

```diff
@@ -694,25 +689,58 @@ export const Visuals = {
   },
 
   drawWitheringBodies() {
-    this.bodiesGraphic ||= this.p.createGraphics(
-      this.windowWidth,
-      this.windowHeight
-    )
+    const { p } = this
+
+    // draw a fake withering body for development
+    // if (this.witheringBodies.length === 0) {
+    //   this.witheringBodies = [{ position: p.createVector(100, 100) }]
+    // }
+
+    this.bodiesGraphic ||= p.createGraphics(this.windowWidth, this.windowHeight)
     this.bodiesGraphic.noStroke()
     for (const body of this.witheringBodies) {
-      // the body should shrink to nothing as HP goes from 0 to -WITHERING_STEPS
-      const witherMultiplier = 1 + body.life / WITHERING_STEPS
-      const radius =
-        (body.radius * 4 + this.radiusMultiplyer) * witherMultiplier
-
-      // render as a white circle
-      this.bodiesGraphic.fill('white')
-      this.bodiesGraphic.ellipse(
-        body.position.x,
-        body.position.y,
-        radius,
-        radius
+      p.push()
+      p.translate(body.position.x, body.position.y)
+      body.witherSteps ||= 0
+      body.witherSteps++
+      if (body.witherSteps > WITHERING_STEPS) {
+        this.witheringBodies = this.witheringBodies.filter((b) => b !== body)
+        continue
+      }
+
+      // the body should shrink to nothing over WITHERING_STEPS
+      const radius = p.map(
+        WITHERING_STEPS - body.witherSteps,
+        0,
+        WITHERING_STEPS,
+        1,
+        30 // start radius
       )
+
+      // the ghost body pulses a little bit, isn't totally round
```

Comment:
this sounds nice. could also be used as shrinking animation to put the body into a star right?